mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -226,6 +226,31 @@ func TestAccComputeInstance_basic5(t *testing.T) {
 	})
 }
 
+func TestAccComputeInstance_basic_resourceManagerTags(t *testing.T) {
+	t.Parallel()
+
+	var instance compute.Instance
+	var instanceName = fmt.Sprintf("tf-test-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeInstance_basic_resourceManagerTags(instanceName),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
You can remove all `Check` funcs, but we'll want to add an import step- see `TestAccComputeInstance_basic1 `, the `computeInstanceImportStep` invocation. As above, this is an old file and comes from a time when we still used `Check`s. Import largely superseded it.

Reply:
ACK. Can we keep them as it is for now? I can inform GCE team, the owner of this file, to refactor it later.

Reply:
The provider codebase is owned by my team. Our guidance for new tests is to use import steps, see https://googlecloudplatform.github.io/magic-modules/docs/how-to/add-handwritten-test/#simple-tests for an example.

Reply:
You can remove the existence check as well- that's also done by the import step.